filecheck/checker.ts

```diff
@@ -185,34 +241,86 @@ export async function checkFile(
         );
         fse.copyFileSync(compressed.destinationPath, filePath);
       } else {
-        const msg = `${filePath} is ${formatSize(
-          sizeBefore
-        )} can be compressed to ${formatSize(
-          sizeAfter
-        )} (${reductionPercentage.toFixed(0)}%)`;
-        console.warn(msg);
-        console.log(
-          "Consider running again with '--save-compression' and run again " +
-            "to automatically save the newly compressed file."
-        );
-        const relPath =
-          process.env.CI === "true"
-            ? path.relative(process.cwd(), filePath)
-            : filePath;
-        const cliCommand = `yarn filecheck "${relPath}" --save-compression`;
-        console.log(`HINT! Type the following command:\n\n\t${cliCommand}\n`);
-        throw new Error(
-          `${filePath} can be compressed by ~${reductionPercentage.toFixed(0)}%`
+        throw new FixableError(
+          `${filePath} is ${formatSize(
+            sizeBefore
+          )} and can be compressed to ${formatSize(
+            sizeAfter
+          )} (${reductionPercentage.toFixed(0)}%)`,
+          `yarn filecheck --save-compression '${getRelativePath(filePath)}'`
```

Comment:
```suggestion
          `yarn filecheck '${getRelativePath(filePath)}'  --save-compression`
```
For whatever reason options belong after the arguments.

Reply:
Why so? Most `--help` pages list options first:

<img width="547" alt="image" src="https://user-images.githubusercontent.com/495429/206431202-61f91ec6-b999-4684-a01a-aca72015105a.png">
<img width="547" alt="image" src="https://user-images.githubusercontent.com/495429/206431233-0b23a7cf-4ade-4885-9db7-c6d99187c5ef.png">
<img width="547" alt="image" src="https://user-images.githubusercontent.com/495429/206431351-42d59df2-0d56-4a2f-888c-fece783f294f.png">

Reply:
Ah, I see: 🤔 

<img width="549" alt="image" src="https://user-images.githubusercontent.com/495429/206431495-eb5eecef-1104-4a64-a460-6f3ff8c7b7c4.png">

Reply:
Fixed in 2f57d6c1c710a273f0b0b552132420c070e7cdc2.